apps/web/src/app/[lang]/dho/[id]/membership/space/create/page.tsx

```diff
@@ -0,0 +1,32 @@
+import { CreateSubspaceForm } from '@hypha-platform/epics';
+import { getDhoPathAgreements } from '../../../agreements/constants';
+import { Locale } from '@hypha-platform/i18n';
+import { createSpaceService } from '@core/space/server';
+
+type PageProps = {
+  params: Promise<{ lang: Locale; id: string }>;
+};
+
+export default async function CreateSubspacePage({ params }: PageProps) {
+  const { lang, id } = await params;
+
+  const spaceService = createSpaceService();
+
+  const spaceFromDb = await spaceService.getBySlug({ slug: id });
+  if (!spaceFromDb) {
+    // Return an error boundary or redirect to a "Not found" page
+    return <div>Space not found</div>;
```

Comment:
Please check the docs: https://nextjs.org/docs/app/api-reference/functions/not-found